server/controllers/home/index.js

```diff
@@ -0,0 +1,22 @@
+const { Location, Review, MainServices } = require('../../database/models');
+const { customError } = require('../errors');
+
+const Home = async (request, response, next) => {
+  try {
+    const locationFromDB = await Location.findAll();
+    const mainServices = await MainServices.findAll();
+    const reviewFromDB = await Review.findAll({
```

Comment:
here we're waiting for locations to get mainServivces after that we get top ten reviews, 
this id take a time while we're don't need to wait for location to get services as it don't depend on
So maybe we can use 
`Promise.all([])` OR `Promise.all Settled([])`

Read more about the difference [Promise all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all)